### DISCUSSION TITLE
Bug when move some icon in the menu application

### DISCUSSION THREAD

**Discussion by FelipeIgansi** [linuxdeepin/developer-center#11068] [Q&A | 问答板块]
I found a bug, but I don't know where I can Crete this bug (the reposiitory). When you try to move some icon from one page of a menu to other page, the menu application don't be refrash and stay showing the same page.
For exemple, I just move one icon from one page to oher page, and the menu don't change.
![image](https://github.com/user-attachments/assets/7cb4bb44-6043-4cd5-ada4-9e606d843046)
![image](https://github.com/user-attachments/assets/58d09e4a-1071-4cc1-995c-30a15c809b85)
![image](https://github.com/user-attachments/assets/37817ad8-d6ce-410e-88bf-5a9745699d57)

PS: The oher mode of the menu is working correcly
![menu application](https://github.com/user-attachments/assets/23e6ea96-b226-4d21-a830-a78d01cdd80f)

**Comment by BLumia** [comment]
Hi,

Which distro are you using, and which version of dde-launchpad are you using?

Thanks!

**Reply by FelipeIgansi** [reply]
Hello,

dde-launchpad 0.7.0-technical-preview
Kernel 6.9.6-amd64-desktop-rolling

**Reply by BLumia** [reply]
> dde-launchpad 0.7.0-technical-preview

Oh sorry you'll need to use `apt policy dde-launchpad`.

**Reply by FelipeIgansi** [reply]
dde-launchpad:
  Installed: 1.0.6
  Candidate: 1.0.6
  Version table:
 *** 1.0.6 500
        500 https://community-packages.deepin.com/beige beige/main amd64 Packages
        100 /usr/lib/dpkg-db/status

**Reply by BLumia** [reply]
Could you also confirm your Qt version? (`apt policy libqt6core6` would be okay)

Qt 6.8 should have this issue fixed but I'm not sure if the update that upgrading system's Qt packages to 6.8 is shipped.

Thanks!

**Reply by FelipeIgansi** [reply]
Hello, 

I see here that my current version of QT is 6.6, Would it be resolved if I update the version?

libqt6core6:
  Installed: 6.6.1+dfsg-5deepin5
  Candidate: 6.6.1+dfsg-5deepin5
  Version table:
 *** 6.6.1+dfsg-5deepin5 500
        500 https://community-packages.deepin.com/beige beige/main amd64 Packages
        100 /usr/lib/dpkg-db/status
